app/interfaces/api/v2/mi/agfs_scheme_ten_claims.rb

```diff
@@ -0,0 +1,44 @@
+module API
+  module V2
+    module MI
+      class AGFSSchemeTenClaims < Grape::API
+        require 'csv'
+        helpers Claims::MI::Queries
+        helpers API::V2::MIHelper
+
+        content_type :csv, 'text/csv; utf-8'
+        default_format :json
+
+        resource :mi, desc: 'MI endpoint' do
+          params do
+            optional :api_key, type: String, desc: 'REQUIRED: The API authentication key of the user'
```

Comment:
**[not a blocker]** Something I've just noticed now, if the description for this field is accurate, we should probably define as such:

```ruby
requires :api_key, type: String, desc: 'REQUIRED: The API authentication key of the user'
```

To enforce that _param_ to actually be mandatory.

Reply:
I seem to remember that there was an issue with validation (of course!) and setting these fields to `requires`, hence why this pattern of `optional= => REQUIRED` all the way through the API :(